src/main/java/ee/tuleva/onboarding/auth/jwt/JwtAuthorizationFilter.java

```diff
@@ -36,30 +39,42 @@ protected void doFilterInternal(
       }
       try {
         if (SecurityContextHolder.getContext().getAuthentication() == null) {
-          AuthenticatedPerson principal =
-              principalService.getFrom(
-                  jwtTokenUtil.getPersonFromToken(jwtToken),
-                  jwtTokenUtil.getAttributesFromToken(jwtToken));
+          TokenType tokenType = jwtTokenUtil.getTypeFromToken(jwtToken);
+          if (tokenType == TokenType.ACCESS) {
```

Comment:
I would create a validation block at the beginning here instead of indenting the whole business logic by 1 level.
For example:
```
if (tokenType != TokenType.ACCESS) {
  return; // do nothing
}
// business logic
```
instead of:
```
if (tokenType == TokenType.ACCESS) {
  // business logic
}
// do nothing
```